src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -724,17 +724,12 @@ class Simpletask : ThemedActivity() {
                         toggle.onOptionsItemSelected(item)
                     }
                 }
-                return true
-            }
-            R.id.search -> {
             }
+            R.id.search -> { }
             R.id.preferences -> startPreferencesActivity()
             R.id.filter -> startFilterActivity()
-            R.id.context_delete -> deleteTasks(TodoList.selectedTasks)
```

Comment:
We already have `val checkedTasks = TodoList.selectedTasks`, so might as well use it.